climada/hazard/trop_cyclone.py

```diff
@@ -901,8 +885,9 @@ def _compute_windfields(
         Output of `tctrack_to_si`. Which data variables are used in the computation of the wind
         speeds depends on the selected model.
     centroids : np.ndarray with two dimensions
-        Each row is a centroid [lat, lon].
-        Centroids that are not within reach of the track are ignored.
+        Each row is a centroid [lat, lon]. Centroids that are not within reach of the track are
+        ignored. Longitudinal coordinates are assumed to be normalized consistently with the
+        longitudinal coordinates in `si_track`.
```

Comment:
This was the reason for the issue that we are fixing here: This function assumes that the lon coordinates are normalized consistently with the lon coordinates in `si_track` - but they were not, in some cases!